puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -80,3 +85,16 @@ def runCicoJobsInParallel(jobs) {
         }
     }
 }
+
+def notifyDiscourse(heading=null) {
+    body = [heading, env.BUILD_URL]
```

Comment:
Same here, let's pass this in instead of relying on magic `env` inside a function.

Reply:
I don't understand what `heading` is here. Makes me think of subject by its naming but that is already being set?

Reply:
Would `bodyHeading` better cover what it's intended to do? `introText`?